DragonSlayer.h

```diff
@@ -1,8 +1,19 @@
 #pragma once
+#include "Character.h"
+#include "AttackItem.h"
+#include <iostream>
```

Comment:
you're not using anything from `iostream` in this header file.  don't include what you don't use.
- [ ] revise